src/extensions/yfm/Math/index.ts

```diff
@@ -37,10 +36,7 @@ export const Math: ExtensionAuto<MathOptions> = (builder, opts) => {
 
     builder.addKeymap(() => ({
         Enter: ignoreIfCursorInsideMathInline, // ignore breaks in math inline
-        Backspace: chainCommands(
-            moveCursorToEndOfMathInline,
```

Comment:
Now, if I press the Backspace key just before a math-inline block, the entire block will be removed from the document. However, previously, pressing the Backspace key would move the cursor within the math-inline block